internal/service/sesv2/dedicated_ip_pool.go

```diff
@@ -98,16 +109,10 @@ func resourceDedicatedIPPoolRead(ctx context.Context, d *schema.ResourceData, me
 	if err != nil {
 		return create.DiagError(names.SESV2, create.ErrActionReading, ResNameDedicatedIPPool, d.Id(), err)
 	}
-	d.Set("pool_name", d.Id())
-
-	poolNameARN := arn.ARN{
-		Partition: meta.(*conns.AWSClient).Partition,
-		Service:   "ses",
-		Region:    meta.(*conns.AWSClient).Region,
-		AccountID: meta.(*conns.AWSClient).AccountID,
-		Resource:  fmt.Sprintf("dedicated-ip-pool/%s", d.Id()),
-	}.String()
-	d.Set("arn", poolNameARN)
+	poolName := aws.ToString(out.DedicatedIpPool.PoolName)
+	d.Set("pool_name", poolName)
+	d.Set("scaling_mode", string(out.DedicatedIpPool.ScalingMode))
```

Comment:
Just out of curiosity, any difference between `aws.ToString()` and the string cast?